config/jobs/kubernetes/sig-node/dynamic-resource-allocation.yaml

```diff
@@ -78,3 +78,32 @@ periodics:
           value: "1"
         - name: GOPATH
           value: /go
+
+  # This job runs the same tests as ci-node-e2e-crio-dra with Containerd 1.7 runtime
+  - name: ci-node-e2e-containerd-1-7-dra
+    interval: 6h
+    annotations:
+      testgrid-dashboards: sig-node-dynamic-resource-allocation
+      testgrid-tab-name: ci-node-e2e-containerd-1-7-dra
+      testgrid-alert-email: eduard.bartosh@intel.com
+    labels:
+      preset-service-account: "true"
+      preset-k8s-ssh: "true"
+    spec:
```

Comment:
`extra_refs` are missing. Without them, the Kubernetes source code will not be available when the job runs.

Reply:
This job is a copy of the `ci-node-e2e-crio-dra`, which seems to work correctly. Why this one shouldn't?